src/main/Interval.js

```diff
@@ -39,6 +39,16 @@ class Interval {
     return this.contains(other.start) && this.contains(other.stop);
   }
 
+  // Expand range to begin and end on multiples of size
+  expand(size: number, zeroBased: boolean): Interval {
+    var minimum = zeroBased ? 0 : 1;
+    var roundDown = x => x - x % size;
```

Comment:
Ok sounds good! I will rename it. Also, I am still not sure on the `zeroBased `parameter. Another option is changing `zeroBased` to an integer, and passing that in as the minimum. So instead you would have:

```
round(size: number, minimum: number): Interval {
    var roundDown = x => x - x % size;
    var newStart = Math.max(minimum, roundDown(this.start)),
        newStop = roundDown(this.stop + size - 1);
    return new Interval(newStart, newStop);
}
```
This is pretty minor, but it could be cleaner.